src/components/FilterSearchToolbar/FilterSearchToolbar.jsx

```diff
@@ -85,8 +85,7 @@ const FilterSearchToolbar = ({
   }
 
   const handleInfoIconClick = (event) => {
-    isHelperTextShowing ? setIsHelperTextShowing(false) : setIsHelperTextShowing(true)
-
+    setIsHelperTextShowing(!isHelperTextShowing)
```

Comment:
There were a lot of this exact change and similar ones to replace ternaries used to call functions instead of to assign a value to a variable due to the linter actually catching this now.